setup.py

```diff
@@ -11,7 +11,12 @@
 
 # Environment-specific dependencies.
 extras = {
-    'jax': ['jax==0.2.14', 'jaxlib==0.1.67', 'dm-haiku==0.0.4', 'optax==0.0.8']
+    'jax': ['jax==0.2.14', 'jaxlib==0.1.67', 'dm-haiku==0.0.4', 'optax==0.0.8'],
+    'torch': ['torch', 'torchvision', 'dgl', 'dgllife'],
+    'tensorflow': [
+        'tensorflow==2.4.*', 'tensorflow_probability==0.12.*',
```

Comment:
Let's go ahead and bump to tf 2.5.* while we're making these updates since it's been out for a month